api/python/quilt3/packages.py

```diff
@@ -869,16 +878,16 @@ def set(self, logical_key, entry=None, meta=None, serialization_location=None, s
             entry(PackageEntry OR string OR object): new entry to place at logical_key in the package.
                 If entry is a string, it is treated as a URL, and an entry is created based on it.
                 If entry is None, the logical key string will be substituted as the entry value.
-                If entry is an object and we know how to serialize it, it will immediately be serialized and written to
-                disk, either to serialization_location or to a location determined by the tempfile library
+                If entry is an object and quilt knows how to serialize it, it will immediately be serialized and written
```

Comment:
Can you note where users can find which file formats are known to Quilt?

Reply:
That's harder than it sounds. I added a function to the `FormatRegistry` that prints out what formats Quilt supports - https://github.com/quiltdata/quilt/pull/1231/files#diff-4c173192a4cd94c4125490ccf3298e37R315. 

It's a bit hacky, but it works - let me know if you think I should beautify it some more.